core/src/main/java/com/dtolabs/rundeck/core/schedule/JobScheduleManager.java

```diff
@@ -87,4 +87,13 @@
      * @return list dead cluster members
      */
     List<String> getDeadMembers(String uuid);
+
+    /**
+     * Schedule a job to run later
+     *
+     * @param data  dataRundeckproClusterGrailsPlugin
+     *
+     * @return true if successful
+     */
+    boolean scheduleRemoteJob(Map data);
```

Comment:
a `default` implementation should be provided for the normal behavior

Reply:
yes, it is here:
https://github.com/rundeck/rundeck/pull/5089/files#diff-c8e0c96ec93bdecafc6f3d719da88211R136

Reply:
i mean use this
```suggestion
    default boolean scheduleRemoteJob(Map data){
        return false;
    }
```

Reply:
got it